tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/QuickstartPlayground.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Playground Tests
+If your code affects Quickstart Playground, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all of these scenarios. These apply to both Windows 10 and Windows 11.
+
+### First Run and First Setup Experience (Machine Has Never Run Quickstart Playground Before and Needs Dependencies Configured)
+1. Launch Dev Home
+1. Navigate to 'Settings -> Experimental features', and validate that Quickstart Playground is off by default. Then turn it on.
+    1. Validate that the Dev Home Azure extension is brought down as part of turning on the feature. When the extension is ultimately installed, you should see a toast notification from the Store.
+1. Navigate to the machine configuration page and you should see a button for the Quickstart Playground feature. Click on that entry. 
+1. Select the Dev Container with OpenAI extension from the ComboBox.
+    1. You should see a ContentDialog with an Adaptive Card guiding you to install any missing dependencies (i.e., Visual Studio Code, Docker Desktop, Remote Containers extension)
+    1. Use the download links to set up any  missing dependencies, accepting all defaults.
+    1. If Docker Desktop was installed, you'll need to reboot the system, manually re-launch Dev Home, and navigate back to the Quickstart Playground page. 
+1. Once all of the application dependencies have been set up, you will see another Adaptive Card for entering in the OpenAI API key. Enter your key into the PasswordTextBox shown to you by the card.
+1. Navigate away from the page and navigate back to validate that there are no additional Adaptive Cards displayed, thereby indicating that setup for the feature is complete.
+
+### Core Feature (Feature is Enabled and Configured)
+1. From the left-hand navigation menu in the application, click on the Machine configuration entry.  
+1. Click on the "Quickstart Playground" entry in the list and choose the Dev Container with OpenAI extension.
+    1. You will see a TextBox where you can enter in a prompt for a project that you’d like to create. 
+    1. Try clicking on a sample prompt. You should see the text entered in the TextBox.   
+1. Click the Generate button. This will use a language model from OpenAI to generate the project.  
+1. Once the operation is completed, you will see a list of files organized by directory that correspond to the sample project generated by the tool.   
+    1. You should be able to click on a file to view a lightweight preview of its contents.  
+    1. You can choose to save the project, launch it in Visual Studio Code, Cancel out of the flow, or start again with a new prompt.  
```

Comment:
Maybe could include the ability to add feedback if desired about the generation output and/or review the AI terms/privacy policies used

Reply:
Good suggestion! I had a verification step for adding feedback but not for the URLs. I've added the latter.